### DISCUSSION TITLE
toc_depth seems to effect internal links in the content.

### DISCUSSION THREAD

**Discussion by michaelcrook** [squidfunk/mkdocs-material#2474] [Q&A] [answered]
I am working on migrating an older documentation set and I wanted to limit the toc on the right panel. When I set the toc depth to 2 in the mkdocs.yml file, my internal linking to header 3/4 sections no longer work. Is there a way to limit the toc depth and preserve linking between sections that are not included in the toc?

**Comment by squidfunk** [comment, marked as answer]
Make sure to update Markdown to the latest version, as this was fixed in a recent version.
